staging/src/k8s.io/sample-controller/OWNERS

```diff
@@ -0,0 +1,7 @@
+approvers:
+- sttts
+- munnerz
+reviewers:
+- gregory-m
+- kargakis
+- nikhita
```

Comment:
Oops, I am not a kubernetes org member. :/

Reply:
Do you need to be an org member to be added here?

Otherwise you can take a look at how to join the org here - https://github.com/kubernetes/community/blob/master/community-membership.md it's quite straightforward and I'm sure there are many that would sponsor you!

Reply:
> Do you need to be an org member to be added here?

Yes.

Thanks, I'll consider applying! :)

Reply:
@munnerz Now I am member. :+1:

Reply:
@munnerz ^^ now you can readd her